src/compas_view2/forms/editform.py

```diff
@@ -168,46 +200,79 @@ def map_number(self, obj, attribute, name=None, layout=None, update_data=True):
 
         # layout.setContentsMargins(0, 0, 0, 0)
         label = QtWidgets.QLabel(name or str(attribute))
-        if isinstance(obj, list) or isinstance(obj, dict):
+        if type(obj) in [list, dict, np.ndarray]:
```

Comment:
why not `isinstance(obj, (list, dict, np.ndarray))`?

Reply:
ha didn't know you can write like that!